output/tar/tar_output.go

```diff
@@ -51,10 +51,9 @@ func (i Output) Apply(rootPath string) <-chan error {
 
 		err = tar.Run()
 		if err != nil {
-			done <- err
+			done <- output.Report{output.UnknownError.Wrap(err).(*errors.Error), def.Output{}}
 			return
 		}
-
 	}()
 	return done
```

Comment:
I propose we either add tests or usage of the report value (there currently is none since this never, well, has had a value to report) so that appears on the radar, or ignore it until it becomes an issue on its own and try to replace it before then.  WDYT?